app/components/PageHeader/PageHeader.tsx

```diff
@@ -1,10 +1,21 @@
+import Image from "next/image";
 import "./PageHeader.css";
+
 const PageHeader = () => {
   return (
     <header>
-      <div className="logo-container">Logo</div>
+      <div className="logo-container">
+        <Image alt="not working" src="logo.svg" width={250} height={51} />
```

Comment:
Should the `alt` be more descriptive? Or is there a reason for the text saying it isn't working?